tests/unit/modules/network/vyos/test_vyos_firewall_rules.py

```diff
@@ -221,7 +221,7 @@ def test_vyos_firewall_v4_rule_sets_rule_merged_01(self):
                                         log="disable",
                                         protocol="icmp",
                                         fragment="match-frag",
-                                        disabled=True,
+                                        disable=True,
```

Comment:
Can we have tests that:
(a) verify that the new key (`disable`) is rendered in facts and not the older one
(b) verify that `disabled` still works in the task.

Reply:
Done